### PR TITLE
DR-291 TSv3 branch consolidation

### PR DESCRIPTION
**Pull request by daveatweaviate**
[issue](https://semi-technology.atlassian.net/browse/DR-291)
[staging](https://dr-291-t-sv-3-branch-consolidatio--tangerine-buttercream-20c32f.netlify.app/developers/weaviate/client-libraries/typescript/typescript-v3)

### CODE REVIEW

**Comment by daveatweaviate** [comment]
@malgamves - I think this captures everything from the merges. It also cleans up a bit here and there. No major changes though. 

If you think it's ok, Let's merge it into the feature branch and any further changes should be pretty small.

File: developers/weaviate/client-libraries/typescript/typescript-v3.md [hunk 1]
```diff
@@ -251,10 +218,19 @@ const result = await myCollection.query.fetchObjects()
 console.log(JSON.stringify(result, null, 2));
 ```
 
+### Node support only 
+
+The gRPC protocol is fast and provides other internal benefits. Unfortunately, gRPC does not support browser-based client development.
+
+The v3 client uses gRPC to connect to your Weaviate instance. The client supports Node.js, server-based development. It does not support browser-based web client development.
+
+To develop a browser-based application, use the v2 client.
```
Comment (malgamves) [inline developers/weaviate/client-libraries/typescript/typescript-v3.md:227]: we should link to the v2 client here.

**Review by malgamves — APPROVED** [review]

**Comment by malgamves** [comment]
Everything looks good! We should be good to merge

File: developers/weaviate/client-libraries/typescript/typescript-v3.md (hunk 1 above)
Comment (daveatweaviate) [reply developers/weaviate/client-libraries/typescript/typescript-v3.md]: Good catch!

**Pull request merged**